### PR TITLE
Change references from passport to @passport-next/passport etc.

### PR DESCRIPTION
**Pull request by guyellis**
Change references from passport to @passport-next/passport etc.

### CODE REVIEW

**Comment by guyellis** [comment]
@rwky how do you want to manage PRs on these repos? Require at least one approver before merging?

File: .gitignore
Comment (rwky) [inline .gitignore]: Why are we ignoring this? I find it quite useful

**Review by rwky — CHANGES_REQUESTED** [review]
At the moment I'd like to review all PRs myself unless they're minor such as typos/small README updates etc.

PRs should also pass tests via travis, coveralls shouldn't go down and the CHANGELOG should be updated/created.

**Comment by rwky** [comment]
p.s. I just added travis etc, so if you rebase master onto this and push again it should run.

File: .gitignore
Comment (guyellis) [reply .gitignore]: I agree. It's very useful. I was going to chat with you about greenkeeper.io in a future issue and I find that keeping package-lock in-sync when using greenkeeper is trickier than vanilla use of greenkeeper. I was thinking that we would start out with the easy use case for greenkeeper and then migrate onto the scenario where we give greenkeeper write access to these repos so that it can update the package-lock file. Hence my reason for wanting to keep it out of the repo until that scenario is complete. WDYT?

**Comment by guyellis** [comment]
Thanks for adding Travis. I was going to create an issue for you to do that. You beat me to it @rwky

File: package.json
Comment (guyellis) [inline package.json:14]: @rwky is this the licensed used for @passport-next?

File: .gitignore
Comment (rwky) [reply .gitignore]: I'd leave it in for now it's useful for travis. I've never used greenkeeper but it does look useful, though I'd want to review it more thoroughly before giving it write access to the repos. The passport modules don't have many deps so manually updating them isn't a massive challenge right now, especially when we're still forking some.

Best to create a new issue for greenkeeper remove the gitignore line.

Comment (guyellis) [reply .gitignore]: @rwky - sounds good - done!

**Comment by rwky** [comment]
Can you squash c4070672947ad3a0174b6fdfba193a55c3d6785b into 729e2196c3e32b2a39c4226291d249746f421f8f 

And squash 9f658142bd0d0537c3931acafe91e4a287264a11 into  4faee263a4fa8b8eaf69cf9bd63a24eef913df3e into 

That'll give us a cleaner commit history.

Travis bailed I've restarted it. Once travis is happy and the commits are squashed this will be good to go.

**Comment by guyellis** [comment]
@rwky - my apologies. I just squashed it all into one commit without thinking. Is this good to merge now?

**Review by rwky — APPROVED** [review]

**Comment by rwky** [comment]
Merged and pushed to NPM! @guyellis thanks for the contributions :)

**Pull request merged**
